content/docs/1.4.0/advanced-resources/deploy/rancher_windows_cluster.md

```diff
@@ -3,21 +3,43 @@ title: Rancher Windows Cluster
 weight: 5
 ---
 
-In this document, you will learn about how to install Longhorn in a Rancher Windows cluster.
+Rancher can provision a Windows cluster with combination of Linux worker nodes and Windows worker nodes.
+For more information on the Rancher Windows cluster, see the official [Rancher documentation](https://rancher.com/docs/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/).
 
-## The characteristic of Rancher Windows Cluster
-Rancher has the ability to provision a cluster that has a mix of Linux worker nodes and Windows worker nodes.
-For more information about Rancher Windows cluster, please refer to the [official Rancher documentation.](https://rancher.com/docs/rancher/v2.x/en/cluster-provisioning/rke-clusters/windows-clusters/)
+In a Rancher Windows cluster, all Linux worker nodes are:
+- Tainted with the taint `cattle.io/os=linux:NoSchedule`
+- Labeled with `kubernetes.io/os:linux`
 
-In a Rancher Windows cluster, all Linux worker nodes are tainted with the taint `cattle.io/os=linux:NoSchedule` and have the label `kubernetes.io/os:linux`
+Follow this document to [Deploy Longhorn With Supported Helm Chart](#deploy-longhorn-with-supported-helm-chart) or [Setup Longhorn Components For Existing Longhorn](#setup-longhorn-components-for-existing-longhorn).
+
+> **Note**: After Longhorn is deployed, you can launch workloads that use Longhorn volumes only on Linux nodes.
+
+## Deploy Longhorn With Supported Helm Chart
+You can update the Helm value `global.cattle.windowsCluster.enbled` to allow Longhorn installation on the Rancher Windows cluster.
+
+When this value is set to `true`, Longhorn will recognize the Rancher Windows cluster node toleration and deploy Longhorn components with the node selector.
```

Comment:
```suggestion
When this value is set to `true`, Longhorn will recognize the Rancher Windows cluster then deploy Longhorn components with the correct node selector and tolerations so that all Longhorn workloads can be launched on Linux nodes only.
```

Reply:
Updated.